<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DNSQuickEdit</class>
 <widget class="QMainWindow" name="DNSQuickEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>173</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>173</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickDNSUpdate</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="domainDropDown">
          <item>
           <property name="text">
            <string>lnxsystems.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>jnvilo.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>maltacentral.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>igamingdb.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>lnxworld.com</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="recordTypeDropDown">
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CNAME</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MX</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ttlDropDown">
        <property name="toolTip">
         <string>TTL </string>
        </property>
        <item>
         <property name="text">
          <string>600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3600</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="priorityDropDown">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Priority </string>
        </property>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="hostEdit">
      <property name="toolTip">
       <string>hostname </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pointsToEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addRecordButton">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add Record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateRecordButton">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add or Update Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
